let's hide email error stacktrace if smtp is not configured

handle below exception and write single error log message instead that smtp is not configured yet.
[2018-10-27 17:14:12{UTC}] [ERROR] [http-apr-8080-exec-1] [c.q.z.s.a.ServiceExceptionAspect] [Got exception when calling [String com.qaprosoft.zafira.services.services.application.EmailService.sendEmail(IEmailMessage, String[])]]
com.qaprosoft.zafira.services.exceptions.IntegrationException: SMTP server connection is refused.
        at com.qaprosoft.zafira.services.services.application.EmailService.sendEmail(EmailService.java:68) ~[zafira-services-2.0.jar:na]
        at com.qaprosoft.zafira.services.services.application.TestRunService.sendTestRunResultsNotification(TestRunService.java:566) [zafira-services-2.0.jar:na]

--- a/sources/zafira-services/src/main/java/com/qaprosoft/zafira/services/services/application/EmailService.java
+++ b/sources/zafira-services/src/main/java/com/qaprosoft/zafira/services/services/application/EmailService.java
@@ -65,7 +65,7 @@ public class EmailService
 
 		if(! settingsService.isConnected(EMAIL))
 		{
-			throw new IntegrationException("SMTP server connection is refused.");
+			return null;
 		}
 
 		final String text = freemarkerUtil.getFreeMarkerTemplateContent(message.getType().getTemplateName(), message);

